pyswmm/output.py

```diff
@@ -649,22 +582,21 @@ def link_attribute(
 
     @output_open_handler
     def system_attribute(
-        self, attribute: Union[int, str], time_index: Union[int, datetime, None] = None
+        self,
+        attribute: shared_enum.SystemAttribute,
+        time_index: Union[int, datetime, None] = None,
     ):
         """
-        At given time, get a particular system attribute.
+         At given time, get a particular system attribute.
 
-        :param attribute: attribute index or name. On of:
-                          air_temp, rainfall, snow_depth, evap_infil_loss, runoff_flow,
-                          dry_weather_inflow, gw_inflow, rdii_inflow, direct_inflow, total_lateral_inflow,
-                          flood_losses, outfall_flows, volume_stored, evap_rate
-        :type attribute: Union[int, str]
-        :param time_index: datetime or simulation index, defaults to None
-        :type time_index: Union[int, datetime, None]
-        :returns: dict of attribute value for system at given timestep
-        :rtype: dict of {"system",value}
+        :param attribute: attribute from swmm.toolkit.shared_enum.SystemAttribute
+         :type attribute: swmm.toolkit.shared_enum.SystemAttribute
```

Comment:
minor comment, can you adjust the spacing in this section?

Reply:
I fixed the indents in link_attribute and system_attribute doc strings